webservices/docs.py

```diff
@@ -267,13 +267,42 @@
 or `/totals` endpoints.
 
 When comparing the totals from reports to line items. the totals will not match unless you
-take out items where `"memoed_subtotal":true`. Memoed items are subtotals of receipts
-that are already accounted for in another schedule a line item.
+only look at items where `"is_individual":true` since the same transaction is in the data
+multipliable ways to explain the way it may move though different committees as an earmark.
 
 For the Schedule A aggregates, such as by_occupation and by_state, include only individual
 contributions, i.e. receipts from line numbers 11AI and 17A. Again, "memoed" items are not
 included to avoid double counting.
 
+### Methodology for determining individual contributions
+
+For receipts over $200 use FEC code line_number to identify individuals.
+
+The line numbers that specify individuals that are automatically included:
+
+|Line number | description
+|---|---
+|10 |Contribution to Independent Expenditure-Only Committees (Super PACs), Political Committees with non-contribution accounts (Hybrid PACs) and nonfederal party "soft money" accounts (1991-2002) from a person (individual, partnership, limited liability company, corporation, labor organization, or any other organization or group of persons)
+|15 | Contribution to political committees (other than Super PACs and Hybrid PACs) from an individual, partnership or limited liability company
+|15E | Earmarked contributions to political committees (other than Super PACs and Hybrid PACs) from an individual, partnership or limited liability company
+|15J | Memo - Recipient committee's percentage of contribution from an individual, partnership or limited liability company given to joint fundraising committee
+|18J | Memo - Recipient committee's percentage of contribution from a registered committee given to joint fundraising committee
+|30, 30T, 31, 31T, 32 | Individual party codes
+
+For receipts under $200:
+We check the following codes and see if there is "earmark" (or a variation) in the
+description of the contribution.
+|Line number | description
+|---|---
+|11Ai | The itemized individual contributions from F3 schedule A
+|12 | Nonfederal other receipt - Levin Account (Line 2)
+|17 | Itemized individual contributions from Form 3P
+|17A | Itemized individual contributions from Form 3P
+|18 | Itemized individual contributions from Form 3P
+
+Of those transactions,[under $200 OR having the codes 11A, 12, 17, 17A, or 18], we then want to exclude earmarks.
```

Comment:
The way the `is_inferred_individual` check is implemented, a receipt is individual if it's < $200, has a line number in [11AI, 12, 17, ...], and is neither memoed nor includes "earmark" in the memo text. Since we're using `and` in the sql, is the `or` here inconsistent?